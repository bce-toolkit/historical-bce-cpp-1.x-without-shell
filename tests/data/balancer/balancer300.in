NO+P=P4O6+N2