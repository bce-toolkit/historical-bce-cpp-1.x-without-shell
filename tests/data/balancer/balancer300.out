6NO+4P=P4O6+3N2